packages/app-bridge/src/types/DocumentSection.ts

```diff
@@ -7,7 +7,7 @@ export type DocumentSectionApi = {
     revision: unknown;
     slug: string;
     sort: number;
-    title: string;
+    title: Nullable<string>;
```

Comment:
I would consider this as a breaking change, as with v3 you will always get `string` as title and with v4 its `string || null`

Reply:
Is it used by blocks? If not, then it is an internal breaking changes as themes are not released

Reply:
Link Chooser handles it already: https://github.com/Frontify/brand-sdk/blob/dev/packages/guideline-blocks-settings/src/components/Link/LinkSelector/DocumentLinks.tsx#L71
And also in the RTE then as it uses the same component

Reply:
Yes but it doesn't matter if the latest version already supports it, as also partner blocks are for sure using an older version of that. So these blocks would break. Also you never know if a partner block uses it in a different way e.g. navigation block or something like this

Reply:
Sounds good, that flew over my mind, thanks!

Reply:
Changes reverted, so they are only present in v4